lib/resty/iprepd.lua

```diff
@@ -0,0 +1,76 @@
+local cjson = require('cjson')
```

Comment:
Is this included with OpenResty or do we need to include this as a dependency?

Reply:
It's included with openresty.